common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -88,3 +92,153 @@ def test_page_has_accomplishments_banner(self):
             }
         ]
         self.assert_events_match(expected_events, actual_events)
+
+
+@attr('shard_5')
+class CertificateProgressPageTest(EventsTestMixin, UniqueCourseTest):
+    """
+    Tests for verifying Certificate info on Progress tab of course page.
+    """
+
+    def setUp(self):
+        super(CertificateProgressPageTest, self).setUp()
+
+        # set same course number as we have in fixture json
+        self.course_info['number'] = "3355358979513794782079645765720179311111"
+
+        test_certificate_config = {
+            'id': 1,
+            'name': 'Certificate name',
+            'description': 'Certificate description',
+            'course_title': 'Course title override',
+            'signatories': [],
+            'version': 1,
+            'is_active': True
+        }
+        course_settings = {'certificates': test_certificate_config}
+
+        self.course_fixture = CourseFixture(
+            self.course_info["org"],
+            self.course_info["number"],
+            self.course_info["run"],
+            self.course_info["display_name"],
+            settings=course_settings
+        )
+
+        self.course_fixture.add_advanced_settings({
+            "cert_html_view_enabled": {"value": "true"}
+        })
+
+        self.course_fixture.add_update(
+            CourseUpdateDesc(date='January 29, 2014', content='Test course update1')
+        )
+
+        self.course_fixture.add_handout('demoPDF.pdf')
```

Comment:
I don't think need to update course handout, tabs chapter etc for testing padding of an element on progress page.